examples/kubernetes/kuma-demo/README.md

```diff
@@ -0,0 +1,235 @@
+Forwarding from 127.0.0.1:8080 -> 8080
+Forwarding from [::1]:8080 -> 8080
+Forwarding from 127.0.0.1:3001 -> 3001
+Forwarding from [::1]:3001 -> 3001
+</code></pre>
+
+Now you can access the application through your web browser at http://localhost:3001.
+
+The items on the front page are pulled from the Elasticsearch service. While the reviews for each item sit within the Redis service. You can query for individual items and look at their reviews. Happy shopping!
+
+10. Now we will port forward the kuma-control-plane so we can access it with `kumactl`
+
+<pre><code>$ kubectl -n kuma-system port-forward <b>{KUMA_CP_POD_NAME}</b> 5681
+Forwarding from 127.0.0.1:5681 -> 5681
+Forwarding from [::1]:5681 -> 5681
+</code></pre>
+
+Please refer to step 5 to copy the correct `{KUMA_CP_POD_NAME}`.
+
+11. Now configure `kumactl` to point towards the control plane address
+
+```
+$ kumactl config control-planes add --name=kuma-app --address=http://localhost:5681
+added Control Plane "kuma-app"
+switched active Control Plane to "kuma-app"
+```
+
+12. You can use `kumactl` to look at the dataplanes in the mesh. You should see three dataplanes:
+
+```
+$ kumactl get dataplanes
+MESH      NAME                             TAGS
+default   es-pkm29                         component=elasticsearch service=elasticsearch.kuma-demo.svc:80
+default   kuma-demo-app-5b8674794f-7r2sf   app=kuma-demo-api pod-template-hash=5b8674794f service=kuma-demo-api.kuma-demo.svc:3001
+default   redis-master-6b88967745-8ct5c    app=redis pod-template-hash=6b88967745 role=master service=redis-master.kuma-demo.svc:6379 tier=backend
+```
+
+13. You can also use `kumactl` to look at the mesh. As shown below, our default mesh does not have mTLS enabled.
+
+```
+$ kumactl get meshes
+NAME      mTLS   DP ACCESS LOGS
+default   off    off
+```
+
+14. Let's enable mTLS and also create a traffic-permission policy. We will set a policy that allows traffic only between node application and our Elasticsearch service. The expected behavior is that the application will no longer be able to access Redis service for reviews.
+
+```
+$ kubectl apply -f kuma-demo-policy.yaml
+Warning: kubectl apply should be used on resource created by either kubectl create --save-config or kubectl apply
+mesh.kuma.io/default configured
+trafficlog.kuma.io/everything created
+trafficpermission.kuma.io/everyone-to-everyone created
+```
+
+15. Using `kumactl`, inspect the mesh again to see if mTLS is enabled:
+
+```
+kumactl get meshes
+NAME      mTLS   DP ACCESS LOGS
+default   on     off
+```
+
+16. You can also get `traffic-permissions` to see that has been applied correctly:
+
+```
+kumactl get traffic-permissions
+MESH      NAME
+default   node-api-to-elasticsearch-only
+```
+
+17. Now try to access the reviews on each item. They will not load because of the traffic-permissions you described in the `kuma-demo-policy.yaml` file. You can inspect the policy we created by using the following `kumactl` command:
```

Comment:
Give instructions (and copy-and-pastable snippet) how to allow `app => redis` traffic

Reply:
added additional step 18 for this